SynLog.pas

```diff
@@ -5342,7 +5346,7 @@ procedure TSynLogFile.ProcessOneLine(LineBeg, LineEnd: PUTF8Char);
   if (fCount<=fLineHeaderCountToIgnore) or (LineEnd-LineBeg<24) then
     exit;
   if fLineLevelOffset=0 then begin
-    if (fCount>50) or not (LineBeg[0] in ['0'..'9']) then
+    if  not (LineBeg[0] in ['0'..'9']) then

```

Comment:
why is fCount>50 wrong?

Reply:
because it limits viewer to 50 events, that for sure.

Reply:
it is expected to have the header available and discovered within the 50 first text files - this won't limit t 50 events; if it is not the case, then the log format is not correct